store/storetest/user_store.go

```diff
@@ -139,15 +139,15 @@ func testUserStoreUpdate(t *testing.T, ss store.Store) {
 	u1 := &model.User{
 		Email: MakeEmail(),
 	}
-	store.Must(ss.User().Save(u1))
+	ss.User().Save(u1)
```

Comment:
Same here `require.Nil(t, err)`, and in all the same changes in this file.

Reply:
Any argument against a synchronous implementation of `Must` for these cases?  Keeps the line count a little lower, and eschews err variable wrangling.

Reply:
Maybe `MustNow` or `SyncMust`?

Reply:
I don't see it, the require approach is clearer and explicit, you see exactly there what is going on and is consistent with the rest of the test assertions.